include/swift/Serialization/SerializedModuleLoader.h

```diff
@@ -68,6 +68,11 @@ struct SerializedModuleBaseName {
   std::optional<std::string> findInterfacePath(llvm::vfs::FileSystem &fs,
                                                ASTContext &ctx) const;
 
+  /// Returns the package-name of the interface file.
+  std::optional<std::string>
+  getPackageNameFromInterfacePath(StringRef interfacePath,
```

Comment:
This sounds a bit like the package name could be extracted from the path itself so rename `getPackageNameFromInterface`.